src/docs/primitives/a-model.md

```diff
@@ -29,3 +29,9 @@ Using scale to increase to size of the loaded tree model:
 ```html
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
```

Comment:
That's only on the dev branch of A-Frame. You can get `.mtl` today with https://github.com/donmccurdy/aframe-obj-loader-component